composer.json

```diff
@@ -14,7 +14,7 @@
   ],
   "require": {
     "xrstf/composer-php52": "~1.0",
-    "lucatume/args": "~1.0.1"
+    "lucatume/args": "0.1.9"
```

Comment:
The latest version of `lucatume/args` is `1.0.1.1` and it is updated on packagist, why the change?

Reply:
Because the last version doesn't work with this plugin, that is the last version that works and not generate errors

Reply:
Ok, thanks for clarifying. 
Would you mind pasting the error output, it seems like the issue should be fixed, not worked around by requiring an older version.

Reply:
I can remember it was an issue with a function that not exists anymore.